diff -Naur surface-dtx-daemon-orig/etc/systemd/surface-dtx-daemon.service surface-dtx-daemon/etc/systemd/surface-dtx-daemon.service
--- surface-dtx-daemon-orig/etc/systemd/surface-dtx-daemon.service	2020-02-03 14:43:24.194020352 -0800
+++ surface-dtx-daemon/etc/systemd/surface-dtx-daemon.service	2020-02-03 14:44:02.681813056 -0800
@@ -6,7 +6,7 @@
 
 [Service]
 Type=simple
-ExecStart=/usr/bin/surface-dtx-daemon
+ExecStart=/usr/bin/env surface-dtx-daemon
 
 [Install]
 WantedBy=multi-user.target
